src/aat/resources/features/F-102/S-580.td.json

```diff
@@ -0,0 +1,78 @@
+{
+  "_guid_": "S-580",
+  "_extends_": "F-102_Test_Data_Base",
+  "title": "Get jurisdictions available to the user",
+  "specs": [
+    "appropriate idam roles but no CCD user profile",
+    "has CREATE as case access parameter",
+    "contains HTTP 200 Ok status code",
+    "contains the list of jurisdictions a user has access to"
+  ],
+  "user": {
+    "username": "befta.caseworker.1.noprofile@gmail.com"
+  },
```

Comment:
this will inherit the password of another user specified in the parent object, which is not good. every user should use its own password env variable.